src/web/components/CommentsLayout.tsx

```diff
@@ -40,6 +39,16 @@ const bottomPadding = css`
     padding-bottom: ${space[2]}px;
 `;
 
+const flexRowWrapper = css`
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    /* Fixes IE 10/11 bug that collapses this container by default: */
+    /* stylelint-disable-next-line property-no-vendor-prefix */
+    -ms-flex-positive: 1;
```

Comment:
This is `flex-grow: 1`. Do we want to be consistent and also use that for non IE browsers too? It might make for a better solution rather than width: 100%